builds/buildcommand.py

```diff
@@ -12,10 +12,67 @@ def __init__(self, command, commandtype, displayName, infile, outfile):
         self.outfile = outfile
         self.displayName = displayName
         self.success = False
-    
+
+    def process_message(self, message, pipeline_configuration):
```

Comment:
This should be moved to the pipeline at some point in future, it's ok for now as the only supported compiler is gcc anyway.